newa/cli.py

```diff
@@ -124,6 +142,15 @@ def cmd_event(ctx: CLIContext, errata_ids: tuple[str, ...]) -> None:
 def cmd_jira(ctx: CLIContext) -> None:
     ctx.enter_command('jira')
 
+    # This is just to provide fake Jira IDs until we can obtain real ones
+    def jira_issue_identifier() -> str:
+        num = 1
+        while True:
+            yield f'NEWA-{num}'
+            num += 1
+
+    jira_id_gen = jira_issue_identifier()
```

Comment:
https://docs.python.org/3/library/itertools.html#itertools.count comes to rescue :)

```python
jira_id_gen = itertools.count(start=1)
```

Reply:
Updated